test/relationshipTemplates.test.ts

```diff
@@ -82,6 +82,24 @@ describe("Template Tests", () => {
         expect(response.isError).toBeTruthy();
         expect(response.error.code).toBe("error.runtime.validation.invalidPropertyValue");
     });
+
+    test("send and receive a personalized template", async () => {
+        const template = await createTemplate(client1, (await client2.account.getIdentityInfo()).result.address);
+
+        const response = await client2.relationshipTemplates.loadPeerRelationshipTemplate({
+            reference: template.truncatedReference
+        });
+        expect(response).toBeSuccessful(ValidationSchema.RelationshipTemplate);
+    });
+
+    test("send and receive a personalized template via token", async () => {
+        const templateToken = await getTemplateToken(client1, (await client2.account.getIdentityInfo()).result.address);
+
+        const response = await client2.relationshipTemplates.loadPeerRelationshipTemplate({
+            reference: templateToken.truncatedReference
+        });
+        expect(response).toBeSuccessful(ValidationSchema.RelationshipTemplate);
+    });
```

Comment:
Can we check as well that an Identity for which the RelationshipTemplate has not been personalized cannot load it?

Reply:
I only wanted to check that the runtime is called correctly, for that the positive tests are enough. Added checks that the result contains the correct personalization though.